src/main/datatypes/MQ2TradeskillDepotType.cpp

```diff
@@ -148,22 +148,22 @@ bool MQ2TradeskillDepotType::GetMember(MQVarPtr VarPtr, const char* Member, char
 			return true;
 		}
 
-//		case TradeskillDepotTypeMethods::DepositItem:
-//		{
-//#if HAS_TRADESKILL_DEPOT
-//			if (pTradeskillDepotWnd->IsVisible() && pActiveBanker)
-//			{
-//				if (pCursorAttachment && pCursorAttachment->Type == eCursorAttachment_Item)
-//				{
-//					CXPoint pt = pTradeskillDepotWnd->pItemList->GetClientRect().CenterPoint();
-//
-//					pTradeskillDepotWnd->pItemList->HandleLButtonDown(pt, 0);
-//					pTradeskillDepotWnd->pItemList->HandleLButtonUp(pt, 0);
-//				}
-//			}
-//#endif
-//			return true;
-//		}
+		case TradeskillDepotTypeMethods::DepositItem:
+		{
+#if HAS_TRADESKILL_DEPOT
+			if (pTradeskillDepotWnd->IsVisible() && pActiveBanker)
+			{
+				if (pCursorAttachment && pCursorAttachment->Type == eCursorAttachment_Item)
+				{
+					CXPoint pt = pTradeskillDepotWnd->pItemList->GetClientRect().CenterPoint();
+					sprintf_s(DataTypeTemp, "%d %d", pt.x, pt.y);
+					MouseTo(pLocalPlayer, DataTypeTemp);
```

Comment:
Use `MoveMouse` instead of formatting a string and calling `MouseTo`

Reply:
Updated